From b532f9177722d38c531cf04356f8541319a3690a Mon Sep 17 00:00:00 2001
From: Andrew Shadura <andrew.shadura@collabora.co.uk>
Date: Wed, 6 Apr 2016 20:41:08 +0200
Subject: [PATCH] Provide appendMessageNoScroll() and
 appendNextMessageNoScroll().

These two functions are required by Empathy. Not having them causes failures
when the first message is about to be displayed.
---
 ubuntu.AdiumMessageStyle/Contents/Resources/Template.html | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/ubuntu.AdiumMessageStyle/Contents/Resources/Template.html b/ubuntu.AdiumMessageStyle/Contents/Resources/Template.html
index 8a64e3e..02cb208 100644
--- a/ubuntu.AdiumMessageStyle/Contents/Resources/Template.html
+++ b/ubuntu.AdiumMessageStyle/Contents/Resources/Template.html
@@ -33,6 +33,12 @@
 		function appendMessage(html) {
 			var shouldScroll = nearBottom();
 			
+			appendMessageNoScroll(html, shouldScroll);
+		}
+			
+		function appendMessageNoScroll(html, shouldScroll) {
+			shouldScroll = shouldScroll || false;
+			
 			//Remove any existing insertion point
 			var insert = document.getElementById("insert");
 			
@@ -65,6 +71,12 @@
 		function appendNextMessage(html){
 			var shouldScroll = nearBottom();
 			
+			appendNextMessageNoScroll(html, shouldScroll);
+		}
+
+		function appendNextMessageNoScroll(html, shouldScroll){
+			shouldScroll = shouldScroll || false;
+			
 			//Locate the insertion point
 			var insert = document.getElementById("insert");
 		
-- 
2.5.0

